judge/models/problem.py

```diff
@@ -271,6 +273,7 @@ def get_editable_problems(cls, user):
             return cls.objects.all()
 
         q = Q(authors=user.profile) | Q(curators=user.profile)
+        q |= Q(is_organization_private=True, organizations__in=user.profile.admin_of.all())
```

Comment:
Since `get_visible_problems` does not use `get_editable_problems`, a similar line to this should be added. Otherwise, `is_accessible_by` and `get_visible_problems` will not be consistent.